mono/metadata/threadpool-worker.h

```diff
@@ -21,6 +21,17 @@ mono_threadpool_worker_request (void);
 gboolean
 mono_threadpool_worker_notify_completed (void);
 
+#if ENABLE_NETCORE
```

Comment:
```suggestion
#ifdef ENABLE_NETCORE
```

Reply:
@lambdageek let me double check, the MetricTest is quite complicated and according to logs all three values had the same values on both coreclr and mono in different steps